src/components/customerDataInfo/CustomerDataInfo.js

```diff
@@ -0,0 +1,36 @@
+import Paper from '@material-ui/core/Paper';
+import Grid from '@material-ui/core/Grid';
+
+import CardCard from './../../assets/images/Club_Card_MM.png';
+import CustomerDataInfoStyles from './CustomerDataInfoStyles';
+
+const CustomerDataInfo = ({customer}) => {
+    const classes = CustomerDataInfoStyles();
+    const customerLable = ['Club card number', 'Club market', 'Newsletter Sign up', 'Member since', 'Mobile no.'];
+
+    return (
+        <>
+            <Paper elevation={3}>
+                <h3 className={classes.paperHeader}>Customer - Club data</h3>
+                <h2 className={classes.customerName}>{`${customer.salutation} ${customer.lastName} ${customer.firstName}`}</h2>
```

Comment:
toti customerii au 'salutation'? Poate ar merita sa faci o verificare inainte sa afisezi ${customer.salutation}.

Reply:
cand faci enroll de new customer, salutation este mandatory. in principiu toti au salutation :)